Python/upLED and upTEMP Controllers/upLED and upTEMP - Python 3 using ctypes.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+
+"""
+Example of C Libraries for TLUP in Python 3 with CTypes
+
+This example will detect all connected upSeries devices.
+It will connect to the first detected device and will perform some steps depending on the device.
+Currently the upLED and the upTEMP are supported in this example.
+"""
+
+import os
+import time
+from ctypes import *
+
+#Loading DLL file.
+lib = cdll.LoadLibrary("C:\Program Files\IVI Foundation\VISA\Win64\Bin\TLUP_64.dll")
+
+#Counting upSeries devices.
+deviceCount = c_uint32()
+lib.TLUP_findRsrc(0,byref(deviceCount))
+if deviceCount.value > 0:
+    print("Number of upSeries devices found: " + str(deviceCount.value))
+else:
+    print("No upSeries devices found.")
+    exit()
+print()
+
+#Reading model name and serial number of the first connected upSeries device.
+modelName = create_string_buffer(256)
+serialNumber = create_string_buffer(256)
+lib.TLUP_getRsrcInfo(0, 0, modelName, serialNumber, 0, 0)
+print("Connecting to this device:")
+print("Model name: ", (modelName.value).decode(), ", Serial number: ", (serialNumber.value).decode())
+print()
+
+#Initializing the first connected upSeries device.
+upName = create_string_buffer(256)
+lib.TLUP_getRsrcName(0, 0, upName)
+upHandle=c_int(0)
+res=lib.TLUP_init(upName.value, 0, 0, byref(upHandle))
+
+#If the device is an upLED, this section is executed.
+if (modelName.value).decode() == "upLED":
+
+    #Make sure the LED is switched off (0 = off, 1 = on).
+    lib.TLUP_switchLedOutput(upHandle,0)
+
+    #Get information about the connected LED.
+    #!!! This only works for Thorlabs' Mounted LEDs.
+    currentSetpoint = c_double()
+    LEDName = create_string_buffer(256)
+    LEDSerialNumber = create_string_buffer(256)
+    LEDCurrentLimit = c_double()
+    LEDForwardVoltage = c_double()
+    LEDWavelength = c_double()
+    
+    lib.TLUP_getLedInfo(upHandle, LEDName, LEDSerialNumber, byref(LEDCurrentLimit),
+                        byref(LEDForwardVoltage), byref(LEDWavelength))
+    print("Connected LED:", (LEDName.value).decode(), ", serial number:", (LEDSerialNumber.value).decode(),", wavelength [nm]:",LEDWavelength.value)
```

Comment:
We should add this to an IF statement that checks if the return from the LEDInfo command is null so the print will be skipped. If null, set the current limit to something else so the code doesn't error.